feature/sessions/src/commonMain/kotlin/io/github/droidkaigi/confsched/sessions/components/TimetableItemDetailContent.kt

```diff
@@ -74,18 +75,22 @@ private fun DescriptionSection(
     var isOverFlow by remember { mutableStateOf(false) }
 
     Column(modifier = Modifier.padding(8.dp)) {
-        SelectionContainer {
-            ClickableLinkText(
-                content = description,
-                regex = "(https)(://[\\w/:%#$&?()~.=+\\-]+)".toRegex(),
-                onLinkClick = onLinkClick,
-                style = MaterialTheme.typography.bodyLarge,
-                maxLines = if (isExpand) Int.MAX_VALUE else 7,
-                overflow = if (isExpand) TextOverflow.Clip else TextOverflow.Ellipsis,
-                onOverflow = {
-                    isOverFlow = it
-                },
-            )
+        provideSelectionContainerCustomContextMenuForDesktop(
```

Comment:
@kilalabu 
Following the comments below, I have changed the name to one that clearly indicates it is intended for the JVM.
This should resolve any concerns preventing the review from proceeding.
https://github.com/DroidKaigi/conference-app-2025/pull/554#discussion_r2327052954